Example/Stripe iOS Example (Simple)/MyAPIClient.swift

```diff
@@ -38,62 +38,21 @@ class MyAPIClient: NSObject, STPBackendAPIAdapter {
                 }
         }
     }
-    
-    @objc func retrieveCustomer(_ completion: @escaping STPCustomerCompletionBlock) {
-        let url = self.baseURL.appendingPathComponent("customer")
-        Alamofire.request(url)
-            .validate(statusCode: 200..<300)
-            .responseJSON { response in
-                switch response.result {
-                case .success(let result):
-                    if let customer = STPCustomer.decodedObject(fromAPIResponse: result as? [String: AnyObject]) {
-                        completion(customer, nil)
-                    } else {
-                        completion(nil, NSError.customerDecodingError)
-                    }
-                case .failure(let error):
-                    completion(nil, error)
-                }
-        }
-    }
-    
-    @objc func selectDefaultCustomerSource(_ source: STPSourceProtocol, completion: @escaping STPErrorBlock) {
-        let url = self.baseURL.appendingPathComponent("customer/default_source")
-        Alamofire.request(url, method: .post, parameters: [
-            "source": source.stripeID,
-            ])
-            .validate(statusCode: 200..<300)
-            .responseString { response in
-                switch response.result {
-                case .success:
-                    completion(nil)
-                case .failure(let error):
-                    completion(error)
-                }
-        }
-    }
-    
-    @objc func attachSource(toCustomer source: STPSourceProtocol, completion: @escaping STPErrorBlock) {
-        let url = self.baseURL.appendingPathComponent("customer/sources")
-        Alamofire.request(url, method: .post, parameters: [
-            "source": source.stripeID,
+
+    func createCustomerKey(withAPIVersion apiVersion: String, completion: @escaping STPJSONResponseCompletionBlock) {
+        let url = URL(string: "https://bg-stripe.runkit.io/591e05608ca4980012451c55/branches/master/ephemeral_keys")!
```

Comment:
yep, this is temporary – we'll update bindings and the example backend before actually shipping